org.qtodotxt.qtodotxt.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Type=Application
```

Comment:
What's the issue with using the appstream desktop file? https://github.com/QTodoTxt/QTodoTxt/blob/master/packaging/Debian/qtodotxt.desktop

Reply:
Use `desktop-file-edit` instead.

Reply:
bump